setup.py

```diff
@@ -9,14 +9,59 @@
 version = __import__("reco_utils.__init__").VERSION
 
 # Get the long description from the README file
-with open(path.join("reco_utils", "README.md"), encoding="utf-8") as f:
+with open("README.md", encoding="utf-8") as f:
     LONG_DESCRIPTION = f.read()
 
 HASH = environ.get("HASH", None)
 if HASH is not None:
     version += ".post" + str(int(time.time()))
 
-name = environ.get("LIBRARY_NAME", "reco_utils")
+DEPENDENCIES = [
```

Comment:
we should not include all these dependencies here, can we  limit this to dependencies only from packages under reco_utils ? also would prefer to have a separate option for [recommenders] which will bring in dependencies needed from algos in the recommenders package

Reply:
Started that work here.
https://github.com/microsoft/recommenders/pull/1322